File: fixtures/scale/php/large/src/Report/TableRenderer.php
<?php

namespace CipherLab\Report;

class TableRenderer
{
    private array $rows = [];

    public function row(array $cells): void
    {
        $this->rows[] = $cells;
    }

    public function render(): string
    {
        return implode("\n", array_map(fn (array $r) => implode('  ', $r), $this->rows));
    }
}

File: fixtures/scale/csharp/large/Report/TableRenderer.cs
using System.Collections.Generic;
using System.Text;

namespace CipherLab.Report
{
    public class TableRenderer
    {
        private readonly List<string[]> _rows = new List<string[]>();

        public void Row(params string[] cells)
        {
            _rows.Add(cells);
        }

        public string Render()
        {
            var sb = new StringBuilder();
            foreach (var row in _rows)
            {
                sb.AppendLine(string.Join("  ", row));
            }
            return sb.ToString().TrimEnd();
        }
    }
}
